bin/cqlsh.py

```diff
@@ -477,16 +480,21 @@ def __init__(self, hostname, port, color=False,
             profiles = {
                 EXEC_PROFILE_DEFAULT: ExecutionProfile(consistency_level=cassandra.ConsistencyLevel.ONE,
                                                        request_timeout=request_timeout,
-                                                       row_factory=ordered_dict_factory,
-                                                       load_balancing_policy=WhiteListRoundRobinPolicy([self.hostname]))
+                                                       row_factory=ordered_dict_factory)
             }
 
-            self.conn = Cluster(contact_points=(self.hostname,), port=self.port, cql_version=cqlver,
+            if cloudconf is None:
+                kwargs['contact_points'] = (self.hostname,)
+                kwargs['port'] = self.port
+                kwargs['ssl_options'] = sslhandling.ssl_settings(hostname, CONFIG_FILE) if ssl else None
+                profiles[EXEC_PROFILE_DEFAULT].load_balancing_policy = WhiteListRoundRobinPolicy([self.hostname])
+
+            self.conn = Cluster(cql_version=cqlver,
                                 auth_provider=self.auth_provider,
-                                ssl_options=sslhandling.ssl_settings(hostname, CONFIG_FILE) if ssl else None,
                                 control_connection_timeout=connect_timeout,
                                 connect_timeout=connect_timeout,
                                 execution_profiles=profiles,
+                                scylla_cloud=cloudconf,
```

Comment:
@fruch How is that supposed to work?
Aren't we building a C* Cluster here?
I'm trying to test a change in `repair_additional_test.py::TestRepairAdditional::test_repair_while_table_is_dropped`
and I get:
```
>               assert re.search(expr, out + err), f"{expr} not found in {out + err}"
E               AssertionError: cf_del0.* not found not found in Traceback (most recent call last):
E                   File "/home/bhalevy/dev/scylla-2/tools/cqlsh/bin/cqlsh.py", line 2701, in <module>
E                     main(*read_options(sys.argv[1:], os.environ))
E                   File "/home/bhalevy/dev/scylla-2/tools/cqlsh/bin/cqlsh.py", line 2642, in main
E                     shell = Shell(hostname,
E                             ^^^^^^^^^^^^^^^
E                   File "/home/bhalevy/dev/scylla-2/tools/cqlsh/bin/cqlsh.py", line 492, in __init__
E                     self.conn = Cluster(cql_version=cqlver,
E                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
E                   File "cassandra/cluster.py", line 1064, in cassandra.cluster.Cluster.__init__
E                 TypeError: __init__() got an unexpected keyword argument 'scylla_cloud'
E                 
E               assert None
E                +  where None = <function search at 0x7ffb910456c0>('cf_del0.* not found', ('' + 'Traceback (most recent call last):\n  File "/home/bhalevy/dev/scylla-2/tools/cqlsh/bin/cqlsh.py", line 2701, in <modu...64, in cassandra.cluster.Cluster.__init__\nTypeError: __init__() got an unexpected keyword argument \'scylla_cloud\'\n'))
E                +    where <function search at 0x7ffb910456c0> = re.search
```

Reply:
the new cqlsh version supposed to work the scylla fork of the driver only.
question here is how did you endup in situation it's not

I might guess it might be the case if running from source, and not from relocatable package (since the relocatable package of cqlsh now continues the our fork of the driver)

Reply:
please raise it on CCM

Reply:
> the new cqlsh version supposed to work only with scylla fork of the driver. question here is how did you endup in situation it's not
> 
> I might guess it might be the case if running from source, and not from relocatable package (since the relocatable package of cqlsh now continues the our fork of the driver)

Then I think it should:
a. document the requirement
b. explicitly assert that to make the requirement clearer and fail early if it is not satisfied by the environment